src/tasks.js

```diff
@@ -0,0 +1,117 @@
+ko.tasks = (function () {
+    var scheduler,
+        taskQueue = [],
+        taskQueueLength = 0,
+        handleOrigin = 0,
+        processingTask,
+        observer;
+
+    if (observer = window['MutationObserver'] || window['WebKitMutationObserver']) {
+        // Chrome 18+, Firefox 14+, IE 11+, Opera 15+, Safari 6+; borrowed from https://github.com/petkaantonov/bluebird
+        scheduler = (function (callback) {
+            var div = document.createElement("div");
+            new observer(callback).observe(div, {attributes: true});
+            return function () { div.classList.toggle("foo"); };
```

Comment:
`classList` is not supported by Opera Mini 8 per http://caniuse.com/#feat=classlist